.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,13 +8,15 @@ to the issue. -->
 
 - [ ] Commits in meaningful sequence and with useful messages.
 - [ ] Tests added or updated when needed.
-- [ ] `CHANGELOG.md` files updated for packages with externally visible changes.  
+- [ ] `CHANGELOG.md` files updated for packages with externally visible changes.
       **NOTE: _New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd)).
 - [ ] Versions updated in `.cabal` and `CHANGELOG.md` files when necessary, according to the
       [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
-- [ ] Version bounds in `.cabal` files updated when necessary.  
+- [ ] Version bounds in `.cabal` files updated when necessary.
       **NOTE: _If bounds change in a cabal file, that package itself must have a version increase._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process)).
 - [x] Code formatted (use `scripts/fourmolize.sh`).
 - [x] Cabal files formatted (use `scripts/cabal-format.sh`).
+- [x] CDDL files are up to date (use `scripts/gen-cddl.sh`)
+- [x] Generated Plutus examples are up to date (use `scripts/gen-plutus.sh`)
```

Comment:
Not sure about these, for the same reason we didn't add the shellcheck check box - based on this conversation: https://github.com/IntersectMBO/cardano-ledger/pull/4951#discussion_r2010757301

In my opinion having the PR check is enough. 

If we're worried that a contributor will see this script failing and won't know how to fix it, I guess the script shows clearly what to do. 
We could potentially write a paragraph in CONTRIBUTING to explain that there is a process involved, but not sure how necessary that is. I imagine whoever modifies the code in a way that requires regenerating, will understand more or less what's this about.

Reply:
Yeah, I guess we could forgo the checkbox for `gen-plutus.sh`, since I doubt anyone who doesn't know what they're doing would change the Plutus scripts. I think it makes sense for `gen-cddl.sh` though, because we change the serializers quite often. I don't think it would hurt to have multiple reminders to run the script before merging.

Reply:
Ok, I have no strong opinion on this one.